docs/book/user-guide/cloud-guide/aws-guide.md

```diff
@@ -0,0 +1,201 @@
+---
+description: A simple guide to create a AWS stack to run your ZenML pipelines
+---
+
+# Connect your AWS account to ZenML
+
+This page aims to quickly set up a minimal production stack on AWS. With just a
+few simple steps you will set up an IAM user with specifically-scoped
+permissions that ZenML can use to authenticate with the relevant AWS resources.
+
+## 1) Set up credentials and local environment
+
+To follow this guide, you need:
+
+- An active AWS account with necessary permissions for AWS S3, SageMaker, and ECR.
+- ZenML [installed](../../getting-started/installation.md)
+- AWS CLI installed and configured with your AWS credentials. You can follow the instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+Once ready, navigate to the AWS console:
+
+1. Choose an AWS region
+In the AWS console, choose the region where you want to deploy your ZenML stack resources. Make note of the region name (e.g. `us-east-1`, `eu-west-2`, etc.) as you will need it in subsequent steps.
+
+1. Create an IAM user
+Create a new IAM user that ZenML will use to access AWS resources:
+
+- Go to the IAM dashboard in the AWS console
+- Click on "Users" in the left sidebar
+- Click "Add user"
+- Enter a user name like zenml-user
+- Select "Programmatic access" under "Access type"
+- Click "Next: Permissions"
+- Click "Attach existing policies directly"
+- Search for and select the following policies:
+  - AmazonS3FullAccess
+  - AmazonECS_FullAccess
+  - AmazonECRFullAccess
+- Click "Next: Tags"
+- Optionally add tags and click "Next: Review"
+- Click "Create user"
+- Click "Download .csv" to save the access key ID and secret access key. You will need these in the next step.
```

Comment:
Like with the GCP guide, I think we really want to use the AWS CLI commands for these since their UI changes too frequently. It's also easier to follow. So I'd replace all these with CLI commands.